audit/types.proto

```diff
@@ -0,0 +1,46 @@
+syntax = "proto3";
+
+package neo.fs.v2.audit;
+
+option go_package = "github.com/nspcc-dev/neofs-api-go/v2/audit/grpc;audit";
+option csharp_namespace = "NeoFS.API.v2.Audit";
+
+import "refs/types.proto";
+
+// DataAuditResult keeps record of conducted Data Audits. The detailed report is
+// generated separately.
+message DataAuditResult {
+  // Epoch number when the Data Audit was conducted
+  uint64 audit_epoch = 1 [json_name = "auditEpoch"];
+
+  // Container under audit
+  neo.fs.v2.refs.ContainerID container_id = 2 [json_name = "containerID"];
+
+  // Public key of the auditing InnerRing node in a binary format
+  bytes public_key = 3 [json_name = "publicKey"];
+
+  // List of Storage Groups that passed audit PoR stage
+  repeated neo.fs.v2.refs.ObjectID pass_sg = 4 [json_name = "passSG"];
+
+  // List of Storage Groups that failed audit PoR stage
+  repeated neo.fs.v2.refs.ObjectID fail_sg = 5 [json_name = "failSG"];
+
+  // Number of objects under audit placed in an optimal way according to the
+  // containers placement policy when checking PoP
+  uint32 hit = 6 [json_name = "hit"];
+
+  // Number of objects under audit placed in suboptimal way according to the
+  // containers placement policy, but still at a satisfactory level when
+  // checking PoP
+  uint32 miss = 7 [json_name = "miss"];
+
+  // Number of objects under audit stored in a way not confirming placement
+  // policy or not found at all when checking PoP
```

Comment:
Maybe we should notice that `hit`, `miss` and `fail` are counters in audit **sample** and it is not precise values? E.g. `Number of objects under audit sample stored ...`